driver-sync/src/test/functional/com/mongodb/fixture/EncryptionFixture.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *
+ */
+
+package com.mongodb.fixture;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.mongodb.ClusterFixture.getEnv;
+
+/**
+ * Helper class for the CSFLE/QE tests.
+ */
+public final class EncryptionFixture {
+
+    private EncryptionFixture() {
+        //NOP
+    }
+
+    public static Map<String, Map<String, Object>> getKmsProviders(final KmsProviderType... kmsProviderTypes) {
+        return new HashMap<String, Map<String, Object>>() {{
```

Comment:
I've reused this approach for readability purposes as it seems less bulky. Since the suggestion is optional, I'll maintain the current implementation. However, I'm open to revisiting this if you feel strongly about it.